File: src/main/java/org/jdc/template/model/webservice/websearch/dto/DtoResponseData.java
package org.jdc.template.model.webservice.websearch.dto;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

import java.util.List;

@JsonObject
public class DtoResponseData {
    @JsonField
    private List<DtoResult> results;
    @JsonField
    private DtoCursor cursor;

    public List<DtoResult> getResults() {
        return results;
    }

    public void setResults(List<DtoResult> results) {
        this.results = results;
    }

    public DtoCursor getCursor() {
        return cursor;
    }

    public void setCursor(DtoCursor cursor) {
        this.cursor = cursor;
    }
}

File: src/main/kotlin/org/jdc/template/model/webservice/websearch/dto/DtoResponseData.kt
package org.jdc.template.model.webservice.websearch.dto

import com.bluelinelabs.logansquare.annotation.JsonField
import com.bluelinelabs.logansquare.annotation.JsonObject

@JsonObject
class DtoResponseData {
    @JsonField
    var results: List<DtoResult>? = null
    @JsonField
    var cursor: DtoCursor? = null
}
